src/server/hotkey_collector.h

```diff
@@ -30,9 +33,13 @@ class internal_collector_base;
 struct detect_hotkey_result
 {
     int coarse_bucket_index = -1;
-    std::string hot_hash_key;
+    std::string hot_hash_key = "";
```

Comment:
Not needed to assign an empty string, default constructor already behave like this.